services/distribution/src/main/java/app/coronawarn/server/services/distribution/runner/TestDataGeneration.java

```diff
@@ -156,15 +164,30 @@ private long getRetentionStartTimestamp() {
         / ONE_HOUR_INTERVAL_SECONDS;
   }
 
+  /**
+   * Either returns the list of all possible visited countries or only current distribution
+   * This ensure that when test generation runs for a country (e.g. FR) all keys in the
+   * distribution will contain FR in the vistied_countries array.
+   */
+  private List<String> generateListOfVisitedCountries(String distributionCountry) {
+    if (random.nextBoolean()) {
+      return List.of("DE", "DK", "FR");
```

Comment:
based on the doc , the distributionCountry param should be part of the list right?

Reply:
Yes, it should. But this is as a placeholder for the future "list of all countries". Since all the tests use either "DE", "DK" or "FR" I left it as this for now. 

Maybe we should add a new configuration in the yaml file that will be this "list of all countries"? A static list of country keys that will be used here and any other places where such list is needed. 

The whole point is that we need to get this list of valid countries from somewhere.

Reply:
I agree we should have environment variable for that, but i guess it can be done outside this PR